epsie/chain/ptchain.py

```diff
@@ -101,6 +105,7 @@ def __init__(self, parameters, model, proposals, betas=1., swap_interval=1,
             # note that pass by reference is required here if setting
             # Tmax=infty
             adaptive_annealer.setup_annealing(self.betas)
+        self._reset_proposals = reset_proposals
```

Comment:
Call this `reset_after_swaps` to avoid confusion with `chain.reset_proposals` and to make it clear this is a setting rather than a method. Also, just store this as an attribute rather than a property. At the moment you can't toggle it off after the class is initialized. No reason to prevent that.